### DISCUSSION TITLE
GFN

### DISCUSSION THREAD

**Discussion by HomingSky** [SnosMe/awakened-poe-trade#984] [Q&A]
Is there any avaliable way to make awakened working with cloud services? I found only one script for it but unfotunatly it didnt update for crucible(or even maybe for earlier leagues idk). For 1.5 year playing poe on gfn i started to hate price checking items through trade.

**Comment by 9Lucky9** [comment]
SnosMe pointed out that to make it work cloud services need to forward a clipboard data, check link below.
https://snosme.github.io/awakened-poe-trade/download#:~:text=%E2%9D%8C%20Not%20compatible%20with%20%22GeForce%20Now%22%20or%20any%20other%20cloud%20gaming%20service%20that%20do%20not%20forward%20clipboard%20data.
